src/__experimental__/components/input/input-sizes.style.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  small: {
+    height: '32px',
+    padding: '8px'
+  },
+  medium: {
+    height: '40px',
+    padding: '11px'
+  },
+  large: {
+    height: '48px',
+    padding: '13px'
+  }
+};
```

Comment:
we could specify size styles as strings:
```javascript
{
  small: `
    min-height: 32px;
    padding-left: 8px;
    padding-right: 8px;
  `,
  medium: `
    min-height: 40px;
    padding-left: 11px;
    padding-right: 11px;
  `,
  large: `
    min-height: 48px;
    padding-left: 13px;
    padding-right: 13px;
  `
};
```
and then just insert whole blocks with ${({ size }) => sizes[size]}

Reply:
In this case they need to remain as JSON format, as I am using these config values in other components (eg. label) which reflects the same padding values.

https://github.com/Sage/carbon/blob/f986289744f8ea2678821746e296529f29814c83/src/__experimental__/components/label/label.style.js#L19

Reply:
we could import Label style into InputPresentation style and add that attributes here with `& ${Label}`. That will allow us to use the label separately without the Input component.